docs/content/direct/core-chart-argocd..md

```diff
@@ -0,0 +1,57 @@
+# Using Argo CD with KubeStellar Core chart
+
+This documents explains how to use KubeStellar Core chart to:
+
+- Deploy an instance of Argo CD using KubeStellar Core chart that automatically lists the WDSes as desitnation clusters
+- Automatically crete Argo CD applications, including scenarios examples

```

Comment:
The first bullet is awkward grammatically. Writing about "chart that automatically ..." suggests there is another chart that does NOT "automatically ...". And the second bullet is also part of the functionality of the one and only chart. And the sentence introducing the bullets already says that this is something that the one and only chart does. So I would rewrite the bullets to something like the following.

- deploy ArgoCD in the KubeFlex hosting cluster;
- register every WDS as a cluster in ArgoCD; and
- create ArgoCD Applications as specified in one of the chart's values.